renderer/components/Channels/ChannelNodeSearch.js

```diff
@@ -79,69 +76,59 @@ class ChannelNodeSearch extends React.PureComponent {
    *
    * @param {string} value Search string
    */
-  handleSearchUpdated = value => {
-    this.updateContactFormSearchQuery(value)
+  const handleSearchUpdated = value => {
+    updateContactFormSearchQueryRef.current(value)
   }
 
-  /**
-   * setFormApi - Store the formApi on the component context to make it available at this.formApi.
-   *
-   * @param {object} formApi Informed formApi
-   */
-  setFormApi = formApi => {
-    this.formApi = formApi
-  }
+  const theme = useContext(ThemeContext)
 
-  render() {
-    const {
-      searchQuery,
-      updateContactFormSearchQuery,
-      filteredNetworkNodes,
-      intl,
-      ...rest
-    } = this.props
+  return (
+    <Box {...rest} theme={theme}>
```

Comment:
@mrfelton I just did it because previously the `{...rest}` piece also included the theme and I wanted to replicate the previous functionality exactly lest I break something. I will remove it if it's not needed.

Reply:
Done.